### PR TITLE
Vue nav

### PR DESCRIPTION
**Pull request by DXCanas**
## Summary

Trying to get the vue-based nav bar included in master, @MCGallaspy has been spearheading this project and has been helping me rebase this branch for our needs. He also set me up by using Ben's hooks system in a boilerplate I'm going to be using to continue nav integration. 

## TODO

- [ ] Have tests been written for the new code?
- [ ] Has documentation been written/updated?
- [ ] New dependencies (if any) added to requirements file
- [ ] Add an entry to CHANGELOG.rst
- [ ] Add yourself it AUTHORS.rst if you don't appear there

## Reviewer guidance

Most of what I've done is just using Vue features -- they shouldn't break anything. Unfortunately, if you've got questions regarding the hooks business, Mike's a better person to ask.

## Issues addressed

* Taking first steps to include vue code
* Creating a navigation bar prototype based off of the designs by Jessica and Whitney

### CODE REVIEW

**Comment by radinamatic** [comment]
Is this the PR that I should take a peek in for a11y review?
@mcg @DXCanas

**Comment by MCGallaspy** [comment]
Wasn't able to get to it today -- will review tomorrow.

**Comment by MCGallaspy** [comment]
@radinamatic I think so, yes. I'll add some comments indicating where you should focus your attention when I review.

File: kolibri/core/assets/src/navigation.vue [hunk 1]
```diff
@@ -62,6 +61,20 @@
     },
   };
 
+=======
```
Comment (MCGallaspy) [inline kolibri/core/assets/src/navigation.vue:64]: Seems to have been included from a merge conflict resolution.

File: kolibri/core/assets/src/core-base.vue [hunk 2]
```diff
@@ -18,33 +12,21 @@
 
 </template>
```
Comment (MCGallaspy) [inline kolibri/core/assets/src/core-base.vue:13]: @radinamatic this template represents the base for a page -- the `main` and `kolibri-nav` tags are application-specific. Main will be filled in with content based on the page you're on (e.g. different for the learn page vs user management). The kolibri-nav template is found in `navigation.vue`.

File: kolibri/core/assets/src/navigation.vue (hunk 1 above)
Comment (MCGallaspy) [inline kolibri/core/assets/src/navigation.vue:64]: More generally this file seems to have gotten into a bad state. Fix up the errors.

**Pull request closed without merging**